source/intro_release_notes/release_notes/resolved_issues_561.rst

```diff
@@ -12,6 +12,7 @@ The following new features has been backported to 5.6.1:
 - XMLRPC calls can report the client IP and PORT, see :ref:`XML-RPC Server Configuration <oned_conf_xml_rpc_server_configuration>` for details.
 - New quotas for VMS allow you to configure limits for VMs "running", see :ref:`quotas <quota_auth>` for more details.
 - Update Host hook triggers to include all possible states, the states are define :ref:`here <hooks>`.
+- Onezone Onezone set should allow temporary zone changes.
```

Comment:
"Onezone Onezone" seems to be a typo
This should point to the onezone man page linked in the documentation